src/renderer/app.jsx

```diff
@@ -177,28 +199,53 @@ export default class App extends React.Component {
       investSettings,
       recentJobs,
       openJobs,
+      openNavIDs,
       activeTab,
       showDownloadModal,
       downloadedNofN,
     } = this.state;
 
     const investNavItems = [];
     const investTabPanes = [];
-    openJobs.forEach((job) => {
+    openNavIDs.forEach((id) => {
+      const job = openJobs[id];
+      let statusSymbol;
+      switch (job.status) {
+        case 'success':
+          statusSymbol = '\u{2705}'; // green check
```

Comment:
Oh this is pretty cool. My first thought would have been to dig out a symbol / icon library for this.

Reply:
I think I went with your suggested emoji in #113 !